README.md

```diff
@@ -53,10 +53,14 @@ which pushes the gem to rubygems.org.  Next write up the release notes: https://
 
 ### Step 2: Update client gems coupled to the models
 
+**NOTE**: You can skip this step if the new release is a patch-level bump only.
```

Comment:
Are you sure? https://github.com/sul-dlss/sdr-api/blob/main/app/controllers/resources_controller.rb#L73-L81

Reply:
@justinlittman the client libraries don't need to be updated, but the consumers need to have `bundle update cocina-models`, because the pin in the clients float at the patch level, there's no need to update them.

Reply:
What @jcoyne said. And that is done in steps 3b and 4.